main/lsp/requests/workspace_symbols.cc

```diff
@@ -58,60 +61,95 @@ vector<unique_ptr<SymbolInformation>> SymbolMatcher::symbolRef2SymbolInformation
     return results;
 }
 
-/** Allow these symbols to act as namespace separators in user queries. */
+/** Is a symbol that can act as a namespace separator in user queries. */
 inline bool canBeLeadingNamespaceSeparator(char ch) {
-    return ch == ':' || ch == '.' || ch == '#' || ch == ' ';
+    return ch == ':' || ch == '.' || ch == '#' || isspace(ch);
+}
+
+/** Is symbol that can skip to the next word boundary within a symbol. */
+inline bool canMatchWordBoundary(char ch) {
+    return isspace(ch);
+}
+
+inline bool isEligibleSymbol(const core::GlobalState &gs, const core::SymbolData &symbolData,
```

Comment:
Nit: `gs` and `symbolData` aren't used at all.

Reply:
Ah...left that in from other variations that were using them to try and be better about skipping unused symbols.  Will remove.